src/styles/main.scss

```
@@ -12,3 +12,23 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+$columnWidth: 200px;
+
+.catalog {
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+
+  $columns-by-breakpoints:
+    '0' repeat(1, $columnWidth),
```

```suggestion
    '0' $columnWidth,
```
Unnecessary function call.